build-script.py

```diff
@@ -244,61 +235,13 @@ def generate_gyb_file(gyb_file: str) -> None:
             pass
 
 
-# Generate the syntax node `.swift` files from `SyntaxNodes.swift.gyb.template`.
-# `destination_dir` is not `None`, the resulting files will be written to
-# `destination_dir/syntax_nodes`, otherwise they will be written to
-# `sources_dir/gyb_generated/syntax_nodes`.
-def generate_syntax_node_template_gyb_files(
-    destination_dir: Optional[str],
-    gyb_exec: str,
-    add_source_locations: bool,
-    verbose: bool
-) -> None:
-    temp_files_dir = tempfile.gettempdir()
-    make_dir_if_needed(temp_files_dir)
-
-    if destination_dir is None:
-        destination_dir = os.path.join(SWIFTSYNTAX_DIR, "gyb_generated")
-
-    template_destination = os.path.join(destination_dir, "syntax_nodes")
-
-    make_dir_if_needed(template_destination)
-    for previous_gyb_gen_file in os.listdir(template_destination):
-        if previous_gyb_gen_file.endswith(".swift"):
-            if previous_gyb_gen_file not in BASE_KIND_FILES.values():
-                check_call(
-                    ["rm", previous_gyb_gen_file],
-                    cwd=template_destination,
-                    verbose=verbose,
-                )
-
-    for base_kind in BASE_KIND_FILES:
-        output_file_name = BASE_KIND_FILES[base_kind]
-
-        gyb_file = os.path.join(
-            SWIFTSYNTAX_DIR, "SyntaxNodes.swift.gyb.template"
-        )
-
-        generate_single_gyb_file(
-            gyb_exec,
-            gyb_file,
-            output_file_name,
-            template_destination,
-            temp_files_dir,
-            add_source_locations,
-            additional_gyb_flags=["-DEMIT_KIND=%s" % base_kind],
-            verbose=verbose,
-        )
-
-
 # If `temp_directories` is True, creates a dictionary that maps every source dir in
 # `source_dirs` to a unique temporary directory.
 # If `temp_directories` is False, it maps each source dir to the corresponding
 # gyb_generated directory.
 def gyb_dir_mapping(temp_directories: bool) -> Dict[str, str]:
     source_dirs = [
         SYNTAXSUPPORT_DIR,
-        SWIFTSYNTAX_DIR,
```

Comment:
@ahoppen the build was failing because of 
```
diff: /home/build-user/swift-syntax/Sources/SwiftSyntax/gyb_generated: No such file or directory
FAIL: Gyb-generated files committed to repository do not match generated ones. Please re-generate the gyb-files using the following command, open a PR to the SwiftSyntax project and merge it alongside the main PR.$ swift-syntax/build-script.py generate-source-code --toolchain /path/to/toolchain.xctoolchain/usr
```

https://ci.swift.org/job/swift-syntax-PR-Linux/1674/console